### ISSUE TITLE
Option to not display icons

### ISSUE THREAD

**Issue by UltraAgent7** [8bitPit/Niagara-Issues#2824]
### Is your feature request related to a problem?

For those of us who want to make our even phones less stimulating.

### Describe the solution you'd like

In the app icons menu, add an option for no icons.

This leads to each app in the app list and favourites not displaying their icon, and only displaying their name.

### Feature Request Procedure

- [X] I have read the [knowledge base article on feature requests](https://help.niagaralauncher.app/article/44-feature-requests) and followed the steps mentioned

**Comment by aurarora** [comment]
A workaround you could use for this is an icon pack, which includes an empty icon.
Personally, often use the icon `Empty` from the [Lines Icon Pack](https://play.google.com/store/apps/details?id=com.natewren.lines).
I do agree though, that a native way to do this would be nice.

**Comment by UltraAgent7** [comment]
Yes, using an empty icon would be viable too if there was a quick way to set one icon for every app.

**Comment by contact-maximux** [comment]
> Yes, using an empty icon would be viable too if there was a quick way to set one icon for every app.

honor, Not any launcher in this world are doing this magic. I love the way you solve a rejected problem in easy way. I hire you.
